variables.tf

```diff
@@ -83,7 +83,7 @@ variable "node_pools" {
       node_labels = optional(map(string), {
         "dominodatalab.com/node-pool" = "default"
       })
-      node_os             = optional(string, "Linux")
+      node_os             = optional(string, "AzureLinux")
```

Comment:
Do we need to validate that the `os_sku` are not mixed?(based on:)
```The Operator does not support a mix of some nodes running Amazon Linux 2 and others running a supported operating system in the same cluster.```

Reply:
I would assume AKS will fail.

Reply:
You can have different skus in a cluster.

Reply:
We aren't using the operator... we are relying on the pre-installed bits.